Courses/_templates/course/document.html.erb

```diff
@@ -122,71 +115,37 @@
       background-color: #4ac084;
   }
 
-    a.button-notebook {
-      display: block;
-      text-align: center;
-      width: 250px;
-      font-size: 18px;
-      margin-top: 20px;
-      font-family: 'Open Sans';
-      height: 55px;
-      border-style: solid;
-      border-width: 0;
-      cursor: pointer;
-      color: white;
-      padding: 15px;
-      font-weight: 300;
-      background-color: #6dcc9c;
-      border-color: #4ac084;
-    }
+  a.button-notebook {
+    display: block;
+    text-align: center;
+    width: 250px;
+    font-size: 18px;
+    margin-top: 20px;
+    font-family: 'Open Sans';
+    height: 55px;
+    border-style: solid;
+    border-width: 0;
+    cursor: pointer;
+    color: white;
+    padding: 15px;
+    font-weight: 300;
+    background-color: #6dcc9c;
+    border-color: #4ac084;
+  }
 
-    .button-notebook a hover {
-        background-color: #4ac084;
-    }
+  .button-notebook a hover {
+    background-color: #4ac084;
+  }
 
   pre.cypher-syntax {
     background-color: #eaf1f8;
   }
 </style>
-<script src="https://neo4j.com/online_training/cypher-gadget/libs/underscore.js"></script>
-<script src="https://neo4j.com/online_training/cypher-gadget/libs/jquery.js"></script>
-<script src="https://neo4j.com//online_training/cypher-gadget/libs/jquery-ui.min.js"></script>
-<script src="https://neo4j.com/online_training/cypher-gadget/libs/jquery.modal.min.js"></script>
```

Comment:
I think we are not using the above scripts and `jquery` is already declared above (https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js).

As a side note, we are using a more than 3 years old version of JQuery! We should probably remove this dependency.